lxc/storage.go

```diff
@@ -646,6 +646,11 @@ func (c *storageCmd) doStoragePoolShow(client *lxd.Client, name string) error {
 		pool.Config["size"] = shared.GetByteSizeString(int64(sz), 0)
 	}
 
+	sz, err = strconv.ParseUint(pool.Config["volume.size"], 10, 64)
+	if err == nil {
+		pool.Config["volume.size"] = shared.GetByteSizeString(int64(sz), 0)
+	}
```

Comment:
Hmm, so what we do for the container keys is that we store the value as it's provided by the user rather than convert it to bytes. We just attempt to convert it to bytes when received to validate it.

And we convert it to bytes whenever we actually need the value as bytes.

That way you don't get into the case where I set something as "10000MB" and have it returned to me as "10GB" by LXD.